rasa/core/policies/ensemble.py

```diff
@@ -1,525 +1,145 @@
-                policy["featurizer"] = featurizer_func(**featurizer_config)
-
-            policy_name = policy.pop("name")
-            try:
-                constr_func = registry.policy_from_module_path(policy_name)
-                try:
-                    policy_object = constr_func(**policy)
-                except TypeError as e:
-                    raise Exception(f"Could not initialize {policy_name}. {e}") from e
-                parsed_policies.append(policy_object)
-            except (ImportError, AttributeError) as e:
-                raise InvalidPolicyConfig(
-                    f"Module for policy '{policy_name}' could not "
-                    f"be loaded. Please make sure the "
-                    f"name is a valid policy."
-                ) from e
-
-        return parsed_policies
-
-    @classmethod
-    def get_featurizer_from_dict(cls, policy: Dict[Text, Any]) -> Tuple[Any, Any]:
-        """Gets the featurizer initializer and its arguments from a policy config."""
-        # policy can have only 1 featurizer
-        if len(policy["featurizer"]) > 1:
-            raise InvalidPolicyConfig(
-                f"Every policy can only have 1 featurizer "
-                f"but '{policy.get('name')}' "
-                f"uses {len(policy['featurizer'])} featurizers."
-            )
-        featurizer_config = policy["featurizer"][0]
-        featurizer_name = featurizer_config.pop("name")
-        featurizer_func = registry.featurizer_from_module_path(featurizer_name)
-
-        return featurizer_func, featurizer_config
-
-    @classmethod
-    def get_state_featurizer_from_dict(
-        cls, featurizer_config: Dict[Text, Any]
-    ) -> Tuple[Callable, Dict[Text, Any]]:
-        # featurizer can have only 1 state featurizer
-        if len(featurizer_config["state_featurizer"]) > 1:
-            raise InvalidPolicyConfig(
-                f"Every featurizer can only have 1 state "
-                f"featurizer but one of the featurizers uses "
-                f"{len(featurizer_config['state_featurizer'])}."
-            )
-        state_featurizer_config = featurizer_config["state_featurizer"][0]
-        state_featurizer_name = state_featurizer_config.pop("name")
-        state_featurizer_func = registry.state_featurizer_from_module_path(
-            state_featurizer_name
-        )
-
-        return state_featurizer_func, state_featurizer_config
-
+        config: Dict[Text, Any],
+        model_storage: ModelStorage,
+        resource: Resource,
+        execution_context: ExecutionContext,
+    ) -> GraphComponent:
+        """Creates a new instance (see parent class for full docstring)."""
+        return cls()
 
-class SimplePolicyEnsemble(PolicyEnsemble):
-    """Default implementation of a `Policy` ensemble."""
+    def __str__(self) -> Text:
+        return f"{self.__class__.__name__}()"
 
     @staticmethod
-    def is_not_in_training_data(
```

Comment:
I think we need to keep this function somewhere as [this](https://github.com/RasaHQ/rasa/blob/d7eacd7e2ed8e413896e967a23d3b94fcd18d85c/rasa/core/test.py#L825) code is using it

Reply:
that function and related code could use some rework I think 😅  -- for now I've just updated the docstr (which was promising the opposite of what happens :D)

Reply:
thanks!